### DISCUSSION TITLE
Support for collections publish from sourcing to consumption(one instance to other instance)

### DISCUSSION THREAD

**Discussion by pallakartheekreddy** [Sunbird-Knowlg/Community#27] [Feature Requests]
@vinukumar-vs @Krishnaj20 

Current, [import](http://docs.sunbird.org/latest/apis/contentapi/index.html#operation/Search%20Content) API supports publishing objectType contents from one instance to other instance(sourcing to consumption).  

We need to publish collection objects from sourcing to consumption. Kindly help with this.

We need this feature because in coKreat BB we have to build a feature which enables users to create collections and publish those collections to consumption. for more details please find below [PRD](https://project-sunbird.atlassian.net/wiki/spaces/SingleSource/pages/2932637697/Workspace+in+Sourcing+Portal)

**Comment by vinukumar-vs** [comment]
@pallakartheekreddy Thanks for sharing the details. We will go through it & update you.
